Themes/default/languages/index.english.php

```diff
@@ -209,7 +209,7 @@
 $txt['welcome_newest_member'] = 'Please welcome %1$s, our newest member.';
 $txt['admin_center'] = 'Administration Center';
 $txt['last_edit_by'] = '<span class="lastedit">Last Edit</span>: %1$s by %2$s';
-$txt['last_edit_reason'] = '<span class="lastedit">Reason</span>: %1$s';
+$txt['last_edit_reason'] = '<span id="reason" class="lastedit">Reason</span>: %1$s';
```

Comment:
I just noticed this, but won't including an id here result in id conflicts in the final document, @Antes?